src/main/java/com/ua/reststream1/controllers/FileController.java

```diff
@@ -0,0 +1,34 @@
+package com.ua.reststream1.controllers;
+
+import com.ua.reststream1.services.FileService;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.multipart.MultipartFile;
+
+@RestController
+public class FileController {
+
+    private final FileService fileService;
+
+    public FileController(FileService fileService) {
+        this.fileService = fileService;
+    }
+
+    @GetMapping("/getMaxValue")
```

Comment:
nitpick: in REST we try not to use verbs in resource naming, as GET/PUT/POST etc. is kinda describing it;  [Best practices](https://restfulapi.net/resource-naming/)